kfp/pipeline_generator/superpipeline/README.md

```diff
@@ -0,0 +1,6 @@
+## Steps to generate a new super pipeline in KFP v1.
+- The super pipeline is a way to execute several transforms with one pipeline. You can find more details in [multi_transform_pipeline.md](../../doc/multi_transform_pipeline.md)
```

Comment:
Please restrict line length to 80 characters:

How about the following  explanation ?
- The super pipeline allows you to execute several transforms within a single pipeline. For more details, refer to [multi_transform_pipeline.md](../../doc/multi_transform_pipeline.md)
- Create a super_pipeline_definitions.yaml file for the required task. You can refer to the example [super_pipeline_definitions.yaml](./super_pipeline_definitions.yaml).
- Execute ./run.sh < super_pipeline_definitions.yaml> <destination_directory>. Here, `super_pipeline_definitions.yaml` is the super pipeline definition file, that you created above, and destination_directory is the directory where the new super pipeline file will be generated.

*__Note__*: The `component_spec_path` is the path to the `kfp_ray_components` folder and depends on where the workflow is compiled.

Reply:
where do we set `component_spec_path` ?

Reply:
The Readme.md updated.
The user can set the `component_spec_path` in the `super_pipeline_definitions.yaml`.